apps/namd/install_namd_openmpi.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#
+# Relevant environmental variables
+#
+# SKU_TYPE = hb, hc or hbv2 (default hbv2)
+# NAMD_SOURCE_TAR_GZ_LOC, location of NAMD_2.13_Source.tar.gz source code (default hpcuser home dir)
+# NAMD_SMP, set to "smp" if you want to build the smp version (default build non smp version)
+# NAMD_MEMOPT, set to "--with-memopt" if you want to build memopt enabled version (default : memopt is disabled) 
+#
+#
+APP_NAME=NAMD
+APP_VERSION=2.13
+SHARED_APP=/avere/apps
```

Comment:
Can you make this an input variable as root is not going to be avere for all installations

Reply:
yes, thanks for spotting this error. I removed the avere reference.